public/src/client/topic/posts.js

```diff
@@ -33,10 +33,12 @@ define('forum/topic/posts', [
 		ajaxify.data.postcount ++;
 		postTools.updatePostCount(ajaxify.data.postcount);
 
-		if (config.usePagination) {
-			onNewPostPagination(data);
-		} else {
-			onNewPostInfiniteScroll(data);
+		if (config.scrollToMyPost) {
```

Comment:
Changing to ajaxify.data is easy, though the way it is now seems to work. I don't understand why this is better (but I'll be honest, I've barely scratched the surface of figuring out how everything fits together).

Also, I've been looking and I have no clue how to make the change in controllers/topics.js or why it's necessary. Also, the change seems to get passed to the client already.